validator/main.go

```diff
@@ -135,6 +140,48 @@ contract in order to activate the validator client`,
 						return nil
 					},
 				},
+				{
+					Name:        "status",
+					Description: `list the validator status for existing validator keys`,
+					Flags: []cli.Flag{
+						flags.KeystorePathFlag,
+						flags.PasswordFlag,
+					},
+					Action: func(cliCtx *cli.Context) error {
+						keystorePath, passphrase, err := accounts.HandleEmptyFlags(cliCtx, false /*confirmPassword*/)
+						if err != nil {
```

Comment:
This is complex logic that should not be in the main.go file. Perhaps create an entrypoint in the accounts/status.go file for this logic

Reply:
https://github.com/prysmaticlabs/prysm/pull/5784/commits/96593f1f1fb085ee19807074d40cc09f62e1b3df